Disable private SLAAC addresses as that would cause dhcpcd to own the routes
by default.

See http://bugs.freebsd.org/bugzilla/show_bug.cgi?id=194485

--- dhcpcd.conf.orig	2016-01-07 16:47:19 UTC
+++ dhcpcd.conf
@@ -35,4 +35,4 @@ option interface_mtu
 require dhcp_server_identifier
 
 # Generate Stable Private IPv6 Addresses instead of hardware based ones
-slaac private
+#slaac private
